server/pulp/server/controllers/repository.py

```diff
@@ -0,0 +1,750 @@
+from gettext import gettext as _
```

Comment:
still need to add notes once an issue is filed. 
1. make PR
2. make issue to reference PR
3. update PR to regerence issue